src/Console/Command/StatusCommand.php

```
@@ -0,0 +1,41 @@
+ * ```
+ *
+ * @author Wilmer Arambula <terabytesoftw@gmail.com>
+ * @since 0.1
+ */
+#[AsCommand(
+    name: 'status',
+    description: 'Displays the status of all scaffold-tracked files relative to their recorded hashes.',
+)]
+final class StatusCommand extends AbstractScaffoldCommand
+{
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$input'. (undefined)

(UnusedFormalParameter)